### ISSUE TITLE
Please: care about your opinions before commenting

### ISSUE THREAD

**Issue by Pomax** [Pomax/Pomax.github.io#9]
# I love comments!

However, i also want you to be committed here: unless you actually care about a topic I'm writing about, and you want me to know about your personal opinion as a position that you hold and that I should be aware of, you probably don't want to leave a throw-away comment on a post I wrote.

That said, if you genuinely have additional or contrary information that I should reasonably modify my posts over —and those things exist, make no mistake; [I could hold a baseless opinion or may have reported factual mistakes](https://github.com/Pomax/Pomax.github.io/issues/10)— then you should definitely let me know, and through me, everyone else who reads the posts I publish.

However, you're not going to gain any fame from comments: they will never show up on the blog's front page, or even on individual posts: your opinions are important, and that's why I won't just let them sit pointlessly on a page. They live here, as github issues.

## Comments are a way for you to engage with me.

If you don't want engagement, you probably don't actually want to leave a comment, you want to tweet about a post I made, maybe with an [@TheRealPomax](https://twitter.com/TheRealPomax) so that I see it fly by in my twitter feed, while knowing you don't actually expect me to engage with you on a specific issue. I'll likely respond, but we'll keep it at that.

So if you still want to leave a comment: excellent! Hit that "new issue" button in the upper right (or log into github first, if you don't see it. Or sign up for a github account first, if you don't have one. None of these things are unreasonable if you truly care about your opinion and want to engage with me on it, so that's another practical filter) and write something you think applies to a post that I published. If you make good points, I'll likely update the post with them, because that's the kind of person I am.

(If you make less than good points, we might go back and forth a little in the issue before I update my posts. If you make shitty points or you're spouting plain nonsense, I'll just close the issue and no one need ever find out)

## TL;DR

if you skipped to this, just [tweet](https://twitter.com/TheRealPomax) at me.

**Comment by Pomax** [comment]
let's see what happens.
